tests/bluechi_test/bluechictl.py

```diff
@@ -149,6 +149,36 @@ def reset_failed(
             expected_result,
         )
 
+    def get_default_target(
+        self,
+        node_name: str = "",
+        check_result: bool = True,
+        expected_result: int = 0,
+    ) -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
+        cmd = f"get-default {node_name}".strip()
```

Comment:
Do we really need to execute `strip()` here?

Reply:
Changed